test/colab/colab-test.mk

```diff
@@ -0,0 +1,31 @@
+.PHONY: colab-test-internal
+colab-test-internal: $(TEST_TOKEN_FILE) $(COLAB_TEST_HOME)/test_import_login_init.ipynb colab-test-pull
+	$(DOCKER_RUN)
+
+.PHONY: colab-test-internal-local
+colab-test-internal-local: $(TEST_TOKEN_FILE) $(COLAB_TEST_HOME)/test_import_login_init.ipynb colab-test-build
+	$(DOCKER_RUN)
+
+.PHONY: colab-test-pull
+colab-test-pull:
+	@docker pull $(DOCKER_IMAGE_NAME)
+
+.PHONY: colab-test-build
```

Comment:
Should not be `PHONY` right? We want to track `$(COLAB_IMAGE_BUILD_STAMP)`

Reply:
We do track it, in the line right underneath:
```
.PHONY: colab-test-build
colab-test-build: $(COLAB_IMAGE_BUILD_STAMP)
```
It's just used as an alias to make the `colab-test-internal-local` command easier to understand, but we could just as well remove `.PHONY` and use `$(COLAB_IMAGE_BUILD_STAMP)` instead, if we wanted to be less verbose. WDYT?

Reply:
I'm ok with keeping it as it is, thanks for explaining it!